### PR TITLE
chore: add workflow which updates Che variables

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

## What does this pull request change?
Add a job that will be used to update Che system variables sections after changes to che.properties in Che Server. The trigger to this job will be provided in a separate workflow in Che Server project
## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/19264

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]
LGTM, +1

Once merged, see followup work in https://github.com/eclipse-che/che-server/pull/156#discussion_r722358801

File: .github/workflows/che-variables-update.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - name: Create Release
+        run: |
+          git config --global user.name "Mykhailo Kuznietsov"
+          git config --global user.email "mkuznets@redhat.com"
+          export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
+
+          BASE_BRANCH=${{ github.event.inputs.branch }}
+          git checkout ${BASE_BRANCH}
+          ./tools/environment_docs_gen.sh
+
```
Comment (themr0c) [inline .github/workflows/che-variables-update.yml:27]: Add also the other script:

```suggestion
          ./tools/checluster_docs_gen.sh
```

Comment (themr0c) [reply .github/workflows/che-variables-update.yml:27]: Oh no really only environment_docs_gen.sh! I should have read the PR description more carefully.

**Review by themr0c — APPROVED** [review]

**Comment by themr0c** [comment]
NB: I cannot really test.

**Pull request merged**